NOTICE.txt

```diff
@@ -309,218 +309,6 @@ WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 OTHER DEALINGS IN THE SOFTWARE.
 
----
-This product relies on playwright-core
```

Comment:
Oops again? Something wrong with our tooling?

Reply:
Ths has consistently failed for a really long time, needs fixed.